ASWF/CONTRIBUTING.md

```diff
@@ -0,0 +1,333 @@
+Contributing to OpenImageIO
+===========================
+
+> NOTE: This is the proposed post-ASWF-move version of CONTRIBUTING. After the
+> project is legally transferred and moved to the new repo, this file will
+> replace the one at the project root.
+>
+> TO DO:
+>
+> - [ ] Update the mail list sign-up page after the mail list moves.
+> - [ ] Update the repo URL
+> - [ ] Double check the security and info email addresses.
+>
+
+Code contributions to OpenImageIO are always welcome, and [nearly 200
+people](https://github.com/OpenImageIO/oiio/blob/master/CREDITS.md) have done
+so over the years.  Please review this document to get a briefing on our
+process.
+
+
+Mail List and Slack
+-------------------
+
+Contributors should be reading the oiio-dev mail list:
+
+* [oiio-dev](https://lists.aswf.io/g/oiio-dev)
+For developers of the OpenImageIO code itself, or users who are really
+interested in the OIIO internals. This is where we mostly discuss the code
+(including bug reports), but are also happy to answer user questions about
```

Comment:
It says we will mostly discuss the code, including bug reports, on the email list. But then later under the Bug Reports section it says we report bugs in github. So this implies we report bugs in github and then do not discuss them there but must discuss on the email list? Why not keep it all in the github issue where it was reported? 

I have the same question for enhancements.